policies/README.md

```diff
@@ -1,5 +1,12 @@
-# Policies
+ Policies
 
 The `aws-operator` needs IAM permissions in order to properly manage tenant
-clusters on AWS. The policy files here are used to setup such IAM permissions.
-See also [our setup docs](https://github.com/giantswarm/docs/blob/25efccb0960bc739f85d4ef9b2043c694aeccbbd/src/content/guides/prepare-aws-account-for-tenant-clusters/index.md#3-permissions-setup).
+clusters on AWS.
+
+The recommended way of setting up the account for aws-operator is using [our
+terraform modules].
+
+If you prefer to do it manually see [our setup docs].
+
+[our setup docs]: https://github.com/giantswarm/docs/blob/25efccb0960bc739f85d4ef9b2043c694aeccbbd/src/content/guides/prepare-aws-account-for-tenant-clusters/index.md#3-permissions-setup
```

Comment:
```suggestion
[our setup docs]: https://github.com/giantswarm/docs/blob/25efccb0960bc739f85d4ef9b2043c694aeccbbd/src/content/guides/prepare-aws-account-for-tenant-clusters/index.md#create-an-iam-role-for-aws-operator-operator-iam-role
```

docs is about "setting up the account for aws-operator", but link to "our setup docs" points to the anchor about attaching policy